params/config.libevm.go

```diff
@@ -231,3 +205,48 @@ func (r *Rules) extraPayload() *pseudo.Type {
 	}
 	return r.extra
 }
+
+func (c *ChainConfig) extraDescription() string {
```

Comment:
The interfaces should instead be defined on `libevm/params.ChainConfigHooks`. This avoids them being optional interfaces, which could be silently mis-implemented. If an implementation doesn't want to use them then they can embed noop hooks.

Do you want to send me a PR? I'm happy to do it if you don't have time, but I want to make sure that I don't hog all libevm work.

Reply:
I'm adopting your recommendation in [29f6067](https://github.com/ava-labs/go-ethereum/pull/11/commits/29f60675ca351bf4c2659cca85c4a126ebef3917), which you can make into a complete PR with tests etc. For now I will continue on pulling `core/vm` in and finding any additional hooks we need.

Reply:
SGTM. So we don't lose track of the required hooks, can you please open issues with feature requests?